stylesheets/components/cards/cards.css

```diff
@@ -0,0 +1,118 @@
+:root {
+    --footer-icon-color: #666666;
+    --card-subtitle-text-color: #757575;
+  }
+.card-container {
+    background-color: var(--base-color);
+    margin: var(--space-lg);
+    border-radius: var(--br-sm);
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    box-shadow: 0 1px 2px grey;
+    width: 20rem;
+  }
+
+  .card-container.badge-container {
+    width: 20rem;
+  }
+  
+  img {
+    max-inline-size: 100%;
+    aspect-ratio: 1;
+    object-fit: cover;
+    object-position: top center;
+    border-radius: var(--br-sm);
+  }
+
+  .fa-times-circle {
```

Comment:
Just a suggestion like not to use fontawesome defined classes bz you may be using the same icon elsewhere which may create difficulty in debugging so you can define a separate class if you are using here for a specific purpose

Reply:
Yes good point. I have addressed this.